tensorflow/java/src/test/java/org/tensorflow/op/ScopeTest.java

```diff
@@ -148,122 +151,166 @@ public void composite() {
     try (Graph g = new Graph();
         Session sess = new Session(g)) {
       Scope s = new Scope(g);
-      Output data = Const.create(s.withName("data"), new int[] {600, 470, 170, 430, 300}).output();
+      Output<Integer> data =
+          Const.create(s.withName("data"), new int[] {600, 470, 170, 430, 300}).output();
 
       // Create a composite op with a customized name
-      Variance var1 = Variance.create(s.withName("example"), data);
+      Variance<Integer> var1 = Variance.create(s.withName("example"), data, Integer.class);
       assertEquals("example/variance", var1.output().op().name());
 
       // Confirm internally added ops have the right names.
       assertNotNull(g.operation("example/squared_deviation"));
       assertNotNull(g.operation("example/Mean"));
-      assertNotNull(g.operation("example/zero"));
+      // assertNotNull(g.operation("example/zero"));
 
       // Same composite op with a default name
-      Variance var2 = Variance.create(s, data);
+      Variance<Integer> var2 = Variance.create(s, data, Integer.class);
       assertEquals("variance/variance", var2.output().op().name());
 
       // Confirm internally added ops have the right names.
       assertNotNull(g.operation("variance/squared_deviation"));
       assertNotNull(g.operation("variance/Mean"));
-      assertNotNull(g.operation("variance/zero"));
+      // assertNotNull(g.operation("variance/zero"));
 
       // Verify correct results as well.
-      Tensor result = sess.runner().fetch(var1.output()).run().get(0);
+      Tensor<Integer> result =
+          sess.runner().fetch(var1.output()).run().get(0).expect(Integer.class);
       assertEquals(21704, result.intValue());
-      result = sess.runner().fetch(var2.output()).run().get(0);
+      result = sess.runner().fetch(var2.output()).run().get(0).expect(Integer.class);
       assertEquals(21704, result.intValue());
     }
   }
 
   // "handwritten" sample operator classes
-  private static final class Const {
-    private final Output output;
+  private static final class Const<T> {
+    private final Output<T> output;
 
-    static Const create(Scope s, Object v) {
-      try (Tensor value = Tensor.create(v)) {
-        return new Const(
+    @SuppressWarnings("unchecked")
```

Comment:
Avoid the `@SuppressWarnings` by making this (and the next function) be:

```java
static Const<Integer> create(Scope s, int v) {
  return create(s, Tensor.create(Integer.class, v));
}
```

?

(Basically, I want to avoid `@SuppressWarnings` from any test/example code, since those often act as inspiration to programmers)

Reply:
Oops, I missed that one. Thanks